ui/src/shared/components/TableGraphFormat.tsx

```diff
@@ -116,16 +136,7 @@ class TableGraphFormat extends PureComponent<Props, State> {
   }
 
   public componentDidUpdate(prevProps: Props) {
-    const updatedProps = _.keys(_.omit(prevProps, 'data')).filter(
-      k => !_.isEqual(this.props[k], prevProps[k])
-    )
-
-    if (
-      this.props.uuid !== prevProps.uuid ||
-      _.includes(updatedProps, 'tableOptions') ||
-      _.includes(updatedProps, 'fieldOptions') ||
-      _.includes(updatedProps, 'timeFormat')
-    ) {
+    if (!areFormatPropertiesEqual(prevProps, this.props)) {
```

Comment:
ah I see now, the `areFormatPropertiesEqual` stuff is just extracted from before.